doc/developers/maintainer.rst

```diff
@@ -107,34 +107,62 @@ under the ``doc/whats_new/`` folder so PRs that target the next version can
 contribute their changelog entries to this file in parallel to the release
 process.
 
-Minor version release
-~~~~~~~~~~~~~~~~~~~~~
+Minor version release and bug-fix release
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
Minor version release (also known as bug-fix release)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
I thought the version numbers' semantic and scheme were:
 - Major release (e.g. `2.0`)
 - Minor release (e.g. `1.3.0`)
 - Bug-fix release (e.g. `1.2.2`)

This is similar to [the `major.minor.micro` example given by PEP 440](https://peps.python.org/pep-0440/#examples-of-compliant-version-schemes).

Is the version numbers' semantic and scheme that scikit-learn is using documented somewhere?